java/app/build.gradle

```diff
@@ -28,7 +32,12 @@ dependencies {
     implementation 'androidx.navigation:navigation-fragment:2.0.0'
     implementation 'androidx.navigation:navigation-ui:2.0.0'
     implementation 'androidx.lifecycle:lifecycle-extensions:2.0.0'
+    def room_version = "2.2.4"
```

Comment:
Some observations:
- the `def` kinda breaks up the dependencies, is it a better practice in gradle to group/move these to the start of the dependencies block?
- we have a `room_version` that defines a version but all other versions are hardcoded; so there's reusability but inconsistency with how the dependencies are defined
- this is using double quotes versus the single quotes that existed

Reply:
great point! I moved all the versions to ext for easier management. 

a couple good reads during research on this topic:
https://developer.android.com/studio/build/gradle-tips.html#configure-project-wide-properties
https://handstandsam.com/2018/02/11/kotlin-buildsrc-for-better-gradle-dependency-management/